examples/with-amp/pages/_document.js

```diff
@@ -0,0 +1,27 @@
+import Document, { Head } from 'next/document'
+import { DOMProperty } from 'react-dom/../lib/ReactInjection'
```

Comment:
Yes, there is an issue with the path resolution, I had to do a hack as you guessed :P more than happy to change it for a more elegant solution